### DISCUSSION TITLE
which one is the best fitted curve for the data I input?

### DISCUSSION THREAD

**Discussion by YmcGW** [phoebe-project/phoebe2#667] [Questions]
I got three fitted curves when I used following code.
![code](https://user-images.githubusercontent.com/118886353/205071291-222eef2e-3186-422f-b58c-6a2ab27e50e2.png)
But I don't know which one is the best, so could you tell me which one is the best fitted curves.

**Comment by kecnry** [comment]
By passing `sample_num=500, sample_mode='3-sigma'`, phoebe will draw 500 random samples from the `emcee_sol` distributions, compute forward models for each drawn sample, and then expose the -3 sigma, median, +3 sigma bounds (in flux) at each time.  If you'd rather store and plot all 500 models (500 might be a bit slow to store and plot depending on how many times/phases your datasets have), you can choose `sample_mode='all'` instead.  If you only want the median exposed and plotted, choose `sample_mode='median'`.

I made a note (https://github.com/phoebe-project/phoebe2-docs/issues/27) to try to improve the documentation and examples for `sample_mode`, I noticed when searching for examples to send you that there really aren't many available - so sorry about that!

-Kyle

**Comment by YmcGW** [comment]
Your reply is really helpful and thank you for your guidance.



---- Replied Message ----
| From | Kyle ***@***.***> |
| Date | 12/01/2022 22:22 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [phoebe-project/phoebe2] which one is the best fitted curve for the data I input? (Discussion #667) |

By passing sample_num=500, sample_mode='3-sigma', phoebe will draw 500 random samples from the emcee_sol distributions, compute forward models for each drawn sample, and then expose the -3 sigma, median, +3 sigma bounds (in flux) at each time. If you'd rather store and plot all 500 models (500 might be a bit slow to store and plot depending on how many times/phases your datasets have), you can choose sample_mode='all' instead. If you only want the median exposed and plotted, choose sample_mode='median'.

I made a note (phoebe-project/phoebe2-docs#27) to try to improve the documentation and examples for sample_mode, I noticed when searching for examples to send you that there really aren't many available - so sorry about that!

-Kyle

**Reply by YmcGW** [reply]
ok
